test/run_tests.sh

```diff
@@ -93,6 +98,11 @@ function run_all_tests {
   run_test python3 "$CDIR/test_ops.py"
   run_downcast_bf16 python3 "$CDIR/test_data_type.py"
   run_use_bf16 python3 "$CDIR/test_data_type.py"
+  run_test python3 "$CDIR/test_xla_backend.py"
```

Comment:
Nit, can we rename the test to `test_torch_distributed_xla_backend`?

Similarly, can we rename other tests to 
```
test_torch_distributed_all_gather_xla_backend.py
test_torch_distributed_all_reduce_xla_backend.py
....
```

Reply:
Done